decidim-admin/db/migrate/20171219154507_add_officialization_to_users.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddOfficializationToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :decidim_users, :officialized_at, :datetime
```

Comment:
Maybe we should index this since we're performing searching on this column

Reply:
Good point, adding it!